pykickstart/options.py

```diff
@@ -257,6 +257,18 @@ def remove_argument(self, arg, **kwargs):
             self._remove_action(candidate)
             self._option_string_actions.pop(arg)
 
+    def deprecate_argument(self, *args, deprecated):
```

Comment:
This method is not mentioned in the CONTRIBUTING file.

Reply:
Shouldn't it be similar to `remove_argument` and accept `arg` instead of `*args`?

Reply:
This might work better as a separate followup PR which would add the new function, use it, and adjust the docs. Without actual usage it's a guess at best.

Reply:
Yeah, that's my plan. I included it here for discussion and so I didn't lose track of it :)

Reply:
> Shouldn't it be similar to `remove_argument` and accept `arg` instead of `*args`?

`*args` makes it easier to deprecate multiple arguments at the same time. Might be useful to also change remove_argument.